### ISSUE TITLE
"Missing initializer in const declaration"

### ISSUE THREAD

**Issue by popeindustries** [rollup/rollup#2897]
- Rollup Version: 1.14.0
- Operating System (or Browser): MacOS 10.14.5
- Node Version: 12.4.0

### How Do We Reproduce?

```js
// In
export class Foo {
  async foo() {
    const something = await window.something();
  }
}

// Out
class Foo {
  async foo() {
    const something;
  }
}
export { Foo };
```
[Repl](https://rollupjs.org/repl?version=1.14.0&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmV4cG9ydCUyMGNsYXNzJTIwRm9vJTIwJTdCJTVDbiU1Q3Rhc3luYyUyMGZvbygpJTIwJTdCJTVDbiU1Q3QlNUN0Y29uc3QlMjBzb21ldGhpbmclMjAlM0QlMjBhd2FpdCUyMHdpbmRvdy5zb21ldGhpbmcoKSUzQiU1Q24lNUN0JTdEJTVDbiU3RCUyMiUyQyUyMmlzRW50cnklMjIlM0F0cnVlJTdEJTVEJTJDJTIyb3B0aW9ucyUyMiUzQSU3QiUyMmZvcm1hdCUyMiUzQSUyMmVzbSUyMiUyQyUyMm5hbWUlMjIlM0ElMjJteUJ1bmRsZSUyMiUyQyUyMmFtZCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyJTIyJTdEJTJDJTIyZ2xvYmFscyUyMiUzQSU3QiU3RCU3RCUyQyUyMmV4YW1wbGUlMjIlM0FudWxsJTdE)

### Expected Behavior
`const something` should have an initializer (right hand side) value.

### Actual Behavior
Node/browser syntax error when parsing because `const` is declared without value.

**Comment by lukastaegert** [comment]
Seems to be the same issue as #2896, looking into it.

**Comment by lukastaegert** [comment]
Fix at #2899

**Comment by lukastaegert** [comment]
Sorry for the inconvenience, should be fixed in 1.14.1 now.
